src/views/product-downloads-view/components/featured-tutorials-section/featured-tutorials-section.module.css

```diff
@@ -0,0 +1,48 @@
+.sectionHeading {
+  color: var(--token-color-neutral-foreground-primary);
+  margin-bottom: 12px;
+}
+
+/*
+TODO: these @media queries come from the numbers defined in src/pages/style.css
+but I wasn't able to get the @media --mobile and @media --tablet queries to work
+like they do in that file. The code below will eventually need to be refactored,
+this is a temporary copy/paste to get the new install view out.
+*/
```

Comment:
Interesting, so the custom media queries aren't working then? Maybe worth adding a ticket to track 👀

Reply:
They weren't working in this spt, so I'm not sure if they're setup incorrectly or what's happening. They are still working for the hide-on-(tablet/mobile) classes that they affect.